package.json

```diff
@@ -40,12 +42,12 @@
     "immutable": "^3.7.6",
     "lodash": "^4.11.0",
     "moment": "^2.12.0",
-    "react": "~15.4.2",
-    "react-dom": "~15.4.2",
-    "react-native": "0.42.0",
-    "react-native-vector-icons": "^4.0.0",
-    "react-navigation": "^1.0.0-beta.9",
-    "react-redux": "^4.4.5",
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0-beta.5",
```

Comment:
why not the actual stable react 16 release ? https://github.com/facebook/react/releases

Reply:
Looks like peer dependency ist still the react 16 beta. Then let's upgrade to react 16 stable as soon as it's support e.g. with RN 50

Reply:
Aye - I was a bit surprised they paired a beta React version with a release RN - but yes, let's update as soon as possible.